Python/Include/Constant.hpp

```diff
@@ -23,9 +24,10 @@ namespace Rover {
     pybind11::class_<Constant<T>>(module, name.c_str())
       .def(pybind11::init<T>())
       .def("generate", &Constant<T>::generate);
-    implicitly_convertible<Constant<T>, Box<pybind11::object>>();
+    pybind11::implicitly_convertible<Constant<T>, Box<T>>();
+    pybind11::implicitly_convertible<Constant<T>, Box<pybind11::object>>();
```

Comment:
If ```T == pybind11::object``` then this will duplicate line 27. Guard against this as ```pybind11``` doesn't check for redundancy.